cmd/influxd/upgrade/config_test.go

```diff
@@ -438,6 +438,7 @@ storage-validate-keys = false
 storage-wal-fsync-delay = "0s"
 tls-cert = "/etc/ssl/influxdb.pem"
 tls-key = ""
+pprof-enabled = true
```

Comment:
Agh, this needs updated to match the new flag name... or I should revert back to `pprof-enabled` for the 2.x flag